src/community/jdbcconfig/src/main/java/org/geoserver/jdbcconfig/internal/ConfigDatabase.java

```diff
@@ -1477,17 +1472,36 @@ public boolean canSort(Class<? extends CatalogInfo> type, String propertyName) {
         return !propertyTypes.isEmpty();
     }
 
-    void clear(Info info) {
+    void clearCache(Info info) {
         if (info instanceof ServiceInfo) {
             // need to figure out how to remove only the relevant cache
             // entries for the service info, like with InfoIdenties below,
             // that will be able to handle new service types.
             disposeServiceCache();
-        }
```

Comment:
Removing this curly brace is breaking the Java compilation?

Reply:
yeah there appear to be some automatic merging errors. I am fixing some unit tests as well and will repush

Reply:
fixed now